### PR TITLE
deprecated old HmacSignature method, adjusted unittest

### PR DESCRIPTION
**Pull request by DjoykeAbyah**
- Deprecated old Hmac validation method 
- updated unit test with new validateHMACSignature method.

Tested by running the unit tests

### CODE REVIEW

File: src/Adyen/Util/HmacSignature.php [hunk 1]
```diff
@@ -9,6 +9,7 @@ class HmacSignature
     const EVENT_CODE = "eventCode";
 
     /**
+     * @deprecated Use validateHMACVSignature with correct parameter order instead
```
Comment (jillingk) [inline src/Adyen/Util/HmacSignature.php:12]:
```suggestion
     * @deprecated Use validateHMACSignature with correct parameter order instead
```
small type

File: src/Adyen/Util/HmacSignature.php [hunk 2]
```diff
@@ -28,6 +29,26 @@ public function validateHMAC(string $hmacKey, string $hmacSign, string $webhook)
...
         return hash_equals($expectedSign, $hmacKey);
     }
+    /**
+     * @param string $hmacKey Can be found in Customer Area
+     * @param string $hmacSign Can be found in the Webhook headers
+     * @param string $webhook The response from Adyen
+     * @return bool
+     * @throws AdyenException
+     */
+    public function validateHMACSignature(string $hmacKey, string $hmacSign, string $webhook): bool
```
Comment (jillingk) [inline src/Adyen/Util/HmacSignature.php:39]: Are you not switching it up now? The code just committed should be the way forward and this method is the "old" wrong way no?

Comment (DjoykeAbyah) [reply src/Adyen/Util/HmacSignature.php:39]: Ah you're right it seems like I've pulled the non updated version!

Comment (jillingk) [reply src/Adyen/Util/HmacSignature.php:40]: Ahhh okay, you got it right. Thought you already rebased on the contribution but you didn't :)

**Review by jillingk — APPROVED** [review]

File: tests/Unit/Util/HmacSignatureTest.php [hunk 3]
```diff
@@ -164,7 +164,7 @@ public function testBankingWebhookHmacValidation()
             . "{\"balanceAccountId\":\"BA3227C223222H5HVKT3H9WLC\"},\"currency\":\"EUR\"}},\"environment\":"
             . "\"test\",\"type\":\"balancePlatform.balanceAccountSweep.updated\"}";
         $hmac = new HmacSignature();
-        $result = $hmac->validateHMAC(
+        $result = $hmac->validateHMACSignature(
```
Comment (acampos1916) [inline tests/Unit/Util/HmacSignatureTest.php:167]: Is it possible to keep both unit tests while the method is being deprecated? And then we remove the unit test once fully gone

Comment (DjoykeAbyah) [reply tests/Unit/Util/HmacSignatureTest.php:167]: Yes ofcourse I'll commit with both tests

**Review by jillingk — APPROVED** [review]

**Pull request merged**